pandas/core/construction.py

```diff
@@ -531,6 +531,11 @@ def _sanitize_ndim(
     elif result.ndim > 1:
         if isinstance(data, np.ndarray):
             raise ValueError("Data must be 1-dimensional")
+        if is_object_dtype(dtype) and isinstance(dtype, ExtensionDtype):
+            # i.e. PandasDtype("O")
+            result = com.asarray_tuplesafe(data, dtype=object)
```

Comment:
could pull this (L536) out to before L534 but can wait for later

Reply:
not clear to me why that would be beneficial